var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -24,62 +24,80 @@
 ##############################################################################
 
 from spack import *
+import platform
 
 
-class Geant4(Package):
+class Geant4(CMakePackage):
     """Geant4 is a toolkit for the simulation of the passage of particles
     through matter. Its areas of application include high energy, nuclear
     and accelerator physics, as well as studies in medical and space
     science."""
 
     homepage = "http://geant4.cern.ch/"
-    url      = "http://geant4.cern.ch/support/source/geant4.10.01.p03.tar.gz"
+    url = "http://geant4.cern.ch/support/source/geant4.10.01.p03.tar.gz"
 
+    version('10.02.p02', '6aae1d0fc743b0edc358c5c8fbe48657')
     version('10.02.p01', 'b81f7082a15f6a34b720b6f15c6289cfe4ddbbbdcef0dc52719f71fac95f7f1c')
     version('10.01.p03', '4fb4175cc0dabcd517443fbdccd97439')
 
-    variant('qt', default=False, description='Enable Qt support')
+    variant('qt', default=True, description='Enable Qt support')
```

Comment:
Qt is a pain to build, which is probably why it was off by default. But I've never used Geant4 before so I can't say how important/recommended Qt is.